### PR TITLE
feat/tdp 3415 composed bucket slice

### PR DESCRIPTION
**Pull request by douglasmik**
Addition of Content Bucket 1 Slice:
- will merge and get design review, which may result in a further ticket to resolve any misalignment with design

https://nidigitalsolutions.jira.com/jira/software/c/projects/TMRX/boards/1600?modal=detail&selectedIssue=TMRX-1092
https://www.figma.com/file/zYjnqWXVY0cFFaJgcSqM5M/Proposed-Slices?node-id=832-851&t=jYk98rerq0KpnCAn-0

### CODE REVIEW

**Review by adamosborne-tnl — APPROVED** [review]

File: packages/ts-newskit/src/slices/content-bucket-1/__tests__/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Render Content Bucket 1 Slice Slice matches snapshot 1`] = `[Function]`;
```
Comment (bibicollins1) [inline packages/ts-newskit/src/slices/content-bucket-1/__tests__/__snapshots__/index.test.tsx.snap:3]: Not sure these snapshots are really doing what we want them to?

File: packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx [hunk 2]
```diff
@@ -63,20 +63,4 @@ describe('Render Article List Item', () => {
       "Harry & Meghan's New Project?"
     );
   });
-
-  it('Article Type should fall back to grey if colour prop is missing', () => {
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx:67]: Is there a reason we removed this test? Is it no longer required functionality?

File: packages/ts-newskit/src/slices/content-bucket-1/__tests__/__snapshots__/index.test.tsx.snap (hunk 1 above)
Comment (douglasmik) [reply packages/ts-newskit/src/slices/content-bucket-1/__tests__/__snapshots__/index.test.tsx.snap:3]: looks like I forgot to invoke the function - will update now

**Comment by bibicollins1** [comment]
Could you link the ticket on the PR? Just so we can compare design?

File: packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx (hunk 2 above)
Comment (douglasmik) [reply packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx:67]: we should be passing colour through as a conditional prop instead - does not need fallback as will inherit colour from parent if not passed

**Comment by bibicollins1** [comment]
Screenshot 2023-04-11 at 09 45 13
I'm not seeing 4 of the articles?

File: packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx (hunk 2 above)
Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/articleList/__tests__/index.test.tsx:67]: Ah cool - thanks!

**Pull request merged**